core/lib/resources/presence/presence_manager.js

```diff
@@ -216,13 +200,7 @@ PresenceManager.prototype._implicitDisconnect = function(socketId, userId,
 
 PresenceManager.prototype.isOnline = function(message) {
   // Return with online, or with sentry, or with unexpired legacy msg
-  return (message.online && (message.sentry ||
-       (message.at && message.at >= (Date.now() - PresenceManager.autoPubTimeout))));
-};
-
-PresenceManager.prototype.isExpired = function(message) {
-  return (message.online && !message.sentry &&
-      message.at && message.at < (Date.now() - PresenceManager.autoPubTimeout));
+  return (message.online && message.sentry);
```

Comment:
Ya. isOnline is now simply message.online. So you might as well use message.online directly.